discordbot/OverworldDiscordBot.py

```diff
@@ -0,0 +1,282 @@
+# Needs a discord bot account and a server with all the mark ids as emojis.
+
+import asyncio
+import io
+import json
+import socket
+import sys
+from functools import lru_cache
+from threading import Thread
+from time import sleep
+from enum import Enum
+from distutils.util import strtobool
+
+import discord
+import pokebase as pb
+from discord.ext import commands
+from discord.utils import get
+from PIL import Image
+
+sys.path.append('../')
+from lookups import Util
+from nxreader import SWSHReader
+# from . import Channels
+
+class Channels(Enum):
+    NoChannel = 0
+    EventNotificationChannelIdForShiny = 1
+    EventNotificationChannelIdForBrilliant = 2
+    EventNotificationChannelIdForMark = 3
+    EventNotificationChannelIdForRareMark = 4
+    NotificationChannelForInfo = 5
+    TextNotificationChannelIdForShiny = 6
+    TextNotificationChannelIdForBrilliant = 7
+    TextNotificationChannelIdForMark = 8
+    TextNotificationChannelIdForRareMark = 9
+
+class OverworldDiscordBot(commands.Bot):
+    def __init__(self):
+        # config to refactor out later
+        # prefix for bot commands
+        bot_prefix = "$"
```

Comment:
having bot_prefix be in config for more customization is nice

Reply:
This prefix gets defined and passed in during construction, but config doesn't get passed in until afterwards (init even sets self.config to a default empty value). How do we set the bot_prefix from the config.json?

Reply:
none of the discord bot commands are used/usable before the config is passed correct? ``super().__init__(command_prefix=bot_prefix)`` could be delayed to only run after the config is passed

Reply:
Done and done. I also simplified configuration while fixing this.